src/main/java/com/mixpanel/android/viewcrawler/ViewSnapshot.java

```diff
@@ -232,6 +234,21 @@ private void addProperties(JsonWriter j, View v)
         }
     }
 
+    // we only do layouts for views with RelativeLayout for now
+    private void addLayouts(JsonWriter j, View v)
```

Comment:
This is simple enough (particularly if we provide them unconditionally) that we might want to inline it